openmc/data/multipole.py

```diff
@@ -434,7 +433,10 @@ def from_hdf5(cls, group_or_filename):
             group = group_or_filename
         else:
             h5file = h5py.File(group_or_filename, 'r')
-            version = h5file['version'].value[0].decode()
+            try:
```

Comment:
@cjosey did you generate the library using old hdf5 interface which writes string as a array in [this commit](https://github.com/mit-crpg/WHOPPER/commit/5727e8721e9e57e3a177793614c97e61c404ec7f)?